model_utils/network.py

```diff
@@ -137,20 +140,32 @@ def bidirectional_simple_rnn_bn_layer(name, input, size, share_weights):
             bias_attr=fluid.ParamAttr(name=name + '_batch_norm_bias'),
             moving_mean_name=name + '_batch_norm_moving_mean',
             moving_variance_name=name + '_batch_norm_moving_variance')
-        #forward and backword in time 
-        forward_rnn = simple_rnn(
-            input=input_proj_bn,
-            size=size,
+        #forward and backword in time
+        forward_cell = RNNCell(
+            hidden_size=size,
+            activation=fluid.layers.brelu,
             param_attr=fluid.ParamAttr(name=name + '_forward_rnn_weight'),
-            bias_attr=fluid.ParamAttr(name=name + '_forward_rnn_bias'),
-            is_reverse=False)
+            bias_attr=fluid.ParamAttr(name=name + '_forward_rnn_bias'))
 
-        reverse_rnn = simple_rnn(
-            input=input_proj_bn,
-            size=size,
+        pad_value = fluid.layers.assign(input=np.array([0.0], dtype=np.float32))
+        input, length = fluid.layers.sequence_pad(input_proj_bn, pad_value)
+        forward_rnn, _ = fluid.layers.rnn(
+            cell=forward_cell, inputs=input, time_major=False, is_reverse=False)
+        forward_rnn = fluid.layers.sequence_unpad(x=forward_rnn, length=length)
+
+        reverse_cell = RNNCell(
+            hidden_size=size,
+            activation=fluid.layers.brelu,
             param_attr=fluid.ParamAttr(name=name + '_reverse_rnn_weight'),
-            bias_attr=fluid.ParamAttr(name=name + '_reverse_rnn_bias'),
+            bias_attr=fluid.ParamAttr(name=name + '_reverse_rnn_bias'))
+        input, length = fluid.layers.sequence_pad(input_proj_bn, pad_value)
```

Comment:
This line is redundant for we have line 151

Reply:
This line has been deleted